app/bundles/FormBundle/Tests/EventListener/ReportSubscriberTest.php

```diff
@@ -60,6 +61,11 @@ class ReportSubscriberTest extends AbstractMauticTestCase
 
     private ReportSubscriber $subscriber;
 
+    /**
+     * @var DncReportService|MockObject
+     */
+    private MockObject $dncReportService;
```

Comment:
Nitpick: We can use this now and it should be an intersection, not one or the other type.
```suggestion
    private MockObject&DncReportService $dncReportService;
```